addons/web_editor/static/lib/odoo-editor/test/utils.js

```diff
@@ -459,18 +459,32 @@ export async function createLink(editor, content) {
     editor.execCommand('createLink', '#', content);
 }
 
-export async function insertText(editor, text) {
+export async function mockInsertCharacter(editor, char) {
+    if(char.length !== 1) {
+        console.error('mockInsertCharacter only mock one character at a time.');
```

Comment:
Couldn't we have kept it as `insertText` and made a loop instead ? In pseudocode:
```
foreach char in text {
    keydown(char);
    input(char);
}
```
Actually, we should check what happens in the DOM when you type very fast. Maybe we should loop on char to trigger multiple keydowns (and we should also trigger keyup) but only trigger one input at the end.